src/data/javascript-quiz.ts

```diff
@@ -1136,7 +1213,19 @@ const javascriptQuiz = [
     Distractor3: "Boolean",
     Explanation:
       "In JavaScript, typeof null is an object.This result of typeof null is actually a bug in the language.",
-    Link: "https://www.freecodecamp.org/news/javascript-typeof-how-to-check-the-type-of-a-variable-or-object-in-js/",
+    Link:
+      "https://www.freecodecamp.org/news/javascript-typeof-how-to-check-the-type-of-a-variable-or-object-in-js/",
+  },
+  {
+    Question: "let a; In JavaScript, what will typeof a return?",
+    Answer: "undefined",
+    Distractor1: "not defined",
+    Distractor2: "",
+    Distractor3: "NaN",
+    Explanation:
+      "In JavaScript, if a variable is been declared and no value is assigned to it, calling property typeof on that variable will return 'undefined'. The global undefined property represents the primitive value undefined. It is one of JavaScript's primitive types",
```

Comment:
Let's shorten this a little bit

```suggestion
      "In JavaScript, if you use the typeof operator on an undeclared variable then it will return undefined. ",
```